apps/lnmarkets/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.7'
+
+services:
+  lnmarkets:
+    image: ghcr.io/lnmarkets/umbrel:v0.0.1
```

Comment:
Is this a multi-architecture docker image? Trying to execute it on my Umbrel gives:

```
standard_init_linux.go:228: exec user process caused: exec format error
```

Reply:
@kibotrel @vafanassieff @RomainRouphael do you have this?

I tried to build the image myself on ARM but it's failing with:


```
[4/4] Building fresh packages...
Done in 103.25s.
Done in 104.90s.
yarn cache v1.22.5
success Cleared cache.
Done in 7.66s.
/usr/local/bin/node-prune: line 1: syntax error: unexpected word (expecting ")")
The command '/bin/sh -c yarn config --silent set cache-folder .yarn &&     yarn workspace api install --frozen-lockfile --production &&     yarn cache clean &&     node-prune &&     rm -f ~/.npmrc &&     rm -f ~/.yarnc &&     rm -f /usr/local/bin/node-prune' returned a non-zero code: 2
```

Reply:
I will update it to support multi arch asap sorry !

Reply:
I added the build plateform for `arm64` tell me if it's okay !

Reply:
Thanks! Testing now.

Reply:
@vafanassieff Image is executing now 🎉

Although I'm getting an LND error because this branch is using an old version of LND and I can't downgrade my LND DB. I've merged the latest upstream changes in to this branch but I can't push them, are you able to give me access rights?